templates/home.php

```diff
@@ -13,7 +13,7 @@
 	<?php
 	global $wp_query;
 
-	WPUtil\Component::render('components/posts-list/posts-list-blog', [
+	get_template_part('components/posts-list/posts-list-blog', [
 		'wp_query_obj' => $wp_query
```

Comment:
Check to make sure `$wp_query` is a `WP_Query` object with `is_a($wp_query, 'WP_Query')`